src/Logging/Logging.Abstractions/src/EventId.cs

```diff
@@ -3,42 +3,80 @@
 
 namespace Microsoft.Extensions.Logging
 {
+    /// <summary>
+    /// Identifies a logging event.
+    /// </summary>
     public readonly struct EventId
     {
+        /// <summary>
+        /// Implicitly creates an EventId from the given <see cref="int"/>.
+        /// </summary>
+        /// <param name="i">The <see cref="int"/> to convert to an EventId.</param>
         public static implicit operator EventId(int i)
         {
             return new EventId(i);
         }
 
+        /// <summary>
+        /// Checks if two specified <see cref="EventId"/>'s have the same value.
```

Comment:
See above comment about confusion on what "matters" here. From the code, it seems two EventIds are equal if their ids are equal. The names are ignored?